src/spec/Schema.php

```diff
@@ -40,7 +40,7 @@
  * @property string[] $required list of required properties
  * @property array $enum
  *
- * @property string $type
+ * @property string|string[] $type
```

Comment:
```suggestion
 * @property string|string[] $type type can only be `string` in OpenAPI 3.0, but can be an array of strings since OpenAPI 3.1
```

we could add a validation for this, but thats out of the scope of this PR.